packages/ipfs-message-port-server/src/core.js

```diff
@@ -231,6 +230,26 @@ class CoreService {
     const content = this.ipfs.cat(location, { offset, length, timeout, signal })
     return encodeCatResult(content)
   }
+
+  /**
+   * @typedef {Object} LsResult
```

Comment:
We should be able to pull the definition for this in from core the same way we're doing in the http client in #3281 right?  Probably in a follow up PR